### PR TITLE
🚒 Add node engines that are supported

### PR DESCRIPTION
**Pull request by rowanc1**
See #1

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -97,6 +97,9 @@
         "ts-jest": "^27.1.4",
         "ts-node": "^10.7.0",
         "typescript": "^4.6.3"
+      },
+      "engines": {
+        "node": ">=14"
```
Comment (rowanc1) [inline npm-shrinkwrap.json:102]: Followed remix package.json here.

File: src/cli/services/utils/utils.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ ${chalk.bold('npm install -g npm@latest')}
 
 async function checkNodeVersion(session: ISession): Promise<boolean> {
   const checking = new Promise<boolean>((resolve) => {
-    check({ node: '>= 14.0.0 <17.0.0', npm: '>=6' }, (error, result) => {
+    check({ node: '>= 14.0.0', npm: '>=6' }, (error, result) => {
```
Comment (rowanc1) [inline src/cli/services/utils/utils.ts:22]: Removed 17 restriction, as it seems to work fine on 18, and added that to the testing.

**Review by rowanc1 — COMMENTED** [review]
Added some notes!

File: src/cli/services/utils/utils.ts (hunk 2 above)
Comment (rowanc1) [reply src/cli/services/utils/utils.ts:22]: See also #102

File: npm-shrinkwrap.json (hunk 1 above)
Comment (stevejpurves) [reply npm-shrinkwrap.json:102]: great, we didn;t have the enginers before -- good to add it 👍🏼

**Review by stevejpurves — APPROVED** [review]

**Pull request merged**